importers/research-importer.php

```diff
@@ -159,6 +159,32 @@ public function get_simple_citation_html( $obj ) {
 			);
 		}
 
+		/**
+		 * Helper function used to return or create a
+		 * WP_Term object for the given taxonomy.
+		 * @author Cadie Stockman
+		 * @since 2.0.0
+		 * @param array $array The array
+		 * @param string $taxonomy_name The taxonomy name to create/assign terms to
+		 * @return string
+		 */
+		public function get_or_create_taxonomy_terms( $array, $taxonomy_name ) {
```

Comment:
Part of the logic for this function exists in the built in [`wp_create_term`](https://developer.wordpress.org/reference/functions/wp_create_term/) wordpress function.